.github/CODEOWNERS

```diff
@@ -5,6 +5,7 @@
 # App
 /x-pack/legacy/plugins/lens/ @elastic/kibana-app
 /x-pack/legacy/plugins/graph/ @elastic/kibana-app
+/src/legacy/server/sample_data/ @elastic/kibana-app
```

Comment:
@TinaHeiligers could you update the path when https://github.com/elastic/kibana/pull/51126 merged?